package-system/AwsIotDeviceSdkCpp/readme.txt

```diff
@@ -26,7 +26,7 @@ void aws_tls_ctx_options_clean_up(struct aws_tls_ctx_options *options) {
   ^
 2 errors generated.
 
-4. The Android version requires 'go' and 'perl' installed.
+4. The Android version is not supported by O3DE currently, but you can try to build it with 'go' and 'perl' installed.
```

Comment:
Mark it as preview
can you provide instruction in list format, and add more details what does last command do?